hooks/boot/30-upgrade.rb

```diff
@@ -4,3 +4,9 @@
   "Run the steps necessary for an upgrade such as migrations, rake tasks, etc.",
   :default => false
 )
+app_option(
+  '--disable-resolve-mismatches',
```

Comment:
Although I guess we break the convention already with `--disable-system-checks`, doesn't really matter to me